README.md

```diff
@@ -220,3 +229,33 @@ This may be better suited for you if you plan to change some of the automaticall
 ## Recap
 
 After following this complete README, we should have an ETH smart contract as well as a Starknet smart contract, both connected to act as a bridge between these two chains.
+
+## Upgrade Contracts in Testnet
+
+### Ethereum
+After deploying the `YABTransfer` contract, you can perform upgrades using the `make ethereum-upgrade` command.
+
+Please note that executing this command will rebuild `YABTransfer.sol`, deploy the new contract to the network, and utilize Foundry to upgrade the proxy contract by changing its implementation to the newly deployed contract.
+
+To execute this action, you must be the **owner** of the contract, and you also need to have set the `YAB_TRANSFER_PROXY_ADDRESS` variable in the `.env` file with the proxy's address.
+
+1. Update the `contracts/solidity/.env` file.
+
+   ```
+      ETH_RPC_URL = Infura or Alchemy RPC URL
+      ETH_PRIVATE_KEY = private key of your ETH wallet
+      ETHERSCAN_API_KEY = API Key to use etherscan to read the Ethereum blockchain
+      SN_MESSAGING_ADDRESS = Starknet Messaging address
+
+      # To upgrade Ethereum contract 
+      YAB_TRANSFER_PROXY_ADDRESS=<YAB_TRANSFER_PROXY_ADDRESS> 
```

Comment:
i would add this new variable to the first time solidity/.env is named (around line 127).
I think this change + automatic update of YAB_TRANSFER_PROXY_ADDRESS will make the .env config much simpler

Reply:
done!